### ISSUE TITLE
[help] change colour for scroll bar

### ISSUE THREAD

**Issue by modmod24** [BoostIO/Boostnote#2261]
hi, the scroll bar on the tags list is very difficult to see, is it possible to apply custom css to it?

How to do it? I could only find class
`SideNav__tagList___browser-main-SideNav`

thank you

**Comment by alexsuslin** [comment]
I guess you can use .SideNav with [pseudo](https://css-tricks.com/almanac/properties/s/scrollbar/)

**Comment by ZeroX-DG** [comment]
Hi, I'm the author of that custom CSS feature and sadly that custom CSS section is only for markdown note previewer not the whole UI. But if you want you can submit a issue for that.

**Comment by modmod24** [comment]
@ZeroX-DG, 
Thanks for confirming this. I was indeed planning to apply custom CSS for the GUI with your feature. 

This is how BoostIO currently looks like:

![boostnote_contrast](https://user-images.githubusercontent.com/25648628/43771460-36b6387e-9a37-11e8-9950-f5cdafcdefc1.png)

You can see the black scroll bar in the black column. It has the color `#425c75`. The background is `#435e79`. It is far too little contrast to the eye. I want to enhance this. 

An immediate fix would be to [use the Interface Theme](https://medium.com/boostnote/boostnote-color-themes-c53cb8e23698) `White`, but I'd rather stick with `Default`. Is it possible to create custom styles for the GUI? And if so can I use `Default` as a basis? I really don't want to change anything but this for now.

**Comment by modmod24** [comment]
If I [add a theme as described here](https://github.com/BoostIO/Boostnote/issues/200#issuecomment-271854423), will it get deleted when upgrading BoostIO?

**Comment by ZeroX-DG** [comment]
@modmod24 Yes it will be deleted if you upgrade Boostnote. Currently there's no way to change the UI style so you will have to stick to your current scroll bar for a while. I'll ask the developer team if I can add a feature for this.

**Comment by modmod24** [comment]
@ZeroX-DG ok no problem. I am not sure which feature fits best for Boostnote. Enhancing the current theme could also be possible, for example `#ccc` for the bar which is used for text colour. I'll leave that to you guys.

**Comment by modmod24** [comment]
the other themes also need some color enhancement. Also, there is a bug. 

# Color theme black

this is the default theme. It has a grey scroll bar on the right side. 
![default_bar](https://user-images.githubusercontent.com/25648628/43932839-2e85ad7a-9c3e-11e8-91af-6d6e717f0ccd.png)

below is the black theme. It has a grey bar with a white background. This is not consistent with other scrollbars. Also, it is not as visible, the white area should not be visible. The same applies to `solarized` and `solarized black`.  
![black1040pix](https://user-images.githubusercontent.com/25648628/43932795-fd70ec9a-9c3d-11e8-8700-911887e4d5a4.png)

Also, note that the `black` theme scrollbar colour changes when you alter the with of the GUI. In the above image, the width ⁼ 1040 pixel, the bar is visible. Note the `margin view`

![black1070pix](https://user-images.githubusercontent.com/25648628/43933039-3d7d35b8-9c3f-11e8-9eaf-9af0bc7344d7.png)

Here, the width = 1070 pixel. The bar is no longer visible. **BUT IT IS IN THE SCREENSHOT!!!** i cannot even show you this, as soon as Boostnote is **not** the active window the bar is visible again. 

-----
# Steps to replicate
1. select theme `dark`
2. create note + content
3. adjust GUI window size such that the scrollbar is visible AND set window width < 1040 pixel
4. expand window width until > 1070 pixel
5. the scrollbar should not be visible anymore
6. make the Boostnote window inactive (click into another application, outside of Boostnote)
7. the scrollbar should be visible again, independent of width

Version: 0.11.8
OS & Version: Ubuntu 16.04

**Comment by ZeroX-DG** [comment]
Can you tell me which version of Boostnote are you using and which OS too?

**Comment by modmod24** [comment]
I am using
OS and Version: Ubuntu 16.04
Version: 0.11.8

**Comment by ZeroX-DG** [comment]
Thank you @modmod24 , I was able to reproduce the problem, PR is comming soon.

**Comment by modmod24** [comment]
text colour for the zoom must be grey in the default theme: 

![zoom_textcolour](https://user-images.githubusercontent.com/25648628/44056384-ff691090-9f3f-11e8-9900-3619d8323517.gif)

-----

OS and Version: Ubuntu 16.04
Version: 0.11.9.pre

**Comment by modmod24** [comment]
I fixed it

-----

before:

![before](https://user-images.githubusercontent.com/25648628/44065947-aa064b06-9f65-11e8-905d-18e978b2edca.gif)

-----

after:
![after](https://user-images.githubusercontent.com/25648628/44065946-a9e25908-9f65-11e8-9180-ee504478491d.gif)
